roles/install_dbserver/tasks/PG_Debian_install.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Install Postgres
+  package:
+    name:
+      - ca-certificates
+      - python3-pycurl
```

Comment:
Same question here about the availability of those python3 packages on Debian9 and Ubuntu18

Reply:
Yes.